VERSION

```diff
@@ -1 +1 @@
-1.18.0
+2.0.0
```

Comment:
Yeah, our tooling for code.stripe.com wants to have a consistent cross-language way of detecting versions, and this is what we use for that.